content/en/docs/tasks/network/extend-service-ip-ranges.md

```diff
@@ -0,0 +1,182 @@
+---
+reviewers:
+- thockin
+- dwinship
+min-kubernetes-server-version: v1.29
+title: Extend Service IP Ranges
+content_type: task
+---
+
+<!-- overview -->
+{{< feature-state state="alpha" for_k8s_version="v1.29" >}}
+
+This document shares how to extend the existing Service IP range assigned to a cluster.
+
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+{{< version-check >}}
+
+<!-- steps -->
+
+## API
+
+Kubernetes clusters that are created with the `MultiCIDRServiceAllocator`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) will create a new ServiceCIDR object with the well-known name `kubernetes` from the kube-apiserver `--service-cluster-ip-range` flag values.
```

Comment:
```suggestion
[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) will create a new ServiceCIDR object that takes the well-known name `kubernetes`, and that uses an IP address range
based on  the value of the `--service-cluster-ip-range` command line argument to kube-apiserver.
```

In the world of IT, flags are for boolean values only; the Golang ecosystem uses “flag” to mean all kinds of command line argument, but this confuses people whose main programming language is not Go.

Reply:
Would be nice to avoid the misleading term “flag”.

Reply:
oh, I thought I applied the suggestion

Reply:
one sec

Reply:
hmm, this file has not the reference to flag, is in the other file

Reply:
Seems sorted!

Reply:
changed all the occurrences of flag